src/main/java/com/github/fge/jsonpatch/OmitOperation.java

```diff
@@ -0,0 +1,54 @@
+package com.github.fge.jsonpatch;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.fasterxml.jackson.databind.node.MissingNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.github.fge.jackson.JsonNumEquals;
+import com.github.fge.jackson.jsonpointer.JsonPointer;
+import com.google.common.base.Equivalence;
+import com.google.common.collect.Iterables;
+
+/**
+ *
+ */
+public class OmitOperation extends PathValueOperation {
+
+    private static final Equivalence<JsonNode> EQUIVALENCE
+        = JsonNumEquals.getInstance();
+    @JsonCreator
+    public OmitOperation(@JsonProperty("path") final JsonPointer path,
+                         @JsonProperty("value") final JsonNode value)
+    {
+        super("omit", path, value);
+    }
+
+    @Override
+    public JsonNode apply(final JsonNode node)
+        throws JsonPatchException
+    {
+        if (path.isEmpty()) {
+            if (EQUIVALENCE.equivalent(node, value)) {
+                return MissingNode.getInstance();
+            } else {
+                return node.deepCopy();
+            }
+        }
+        if (path.path(node).isMissingNode())
+            throw new JsonPatchException(BUNDLE.getMessage(
+                "jsonPatch.noSuchPath"));
+
+        final JsonNode ret = node.deepCopy();
+        if (EQUIVALENCE.equivalent(path.path(ret), value)) {
+            final JsonNode parent = path.parent().get(ret);
+            final String rawToken = Iterables.getLast(path).getToken().getRaw();
+            if (parent.isObject())
```

Comment:
Yes, I believe so.  If the target is a primitive like string, number, boolean, null, then any JSON pointer other than "" will result in MissingNode.  It follows the pattern of RemoveOperation.

It is not explicit in the spec, but this line defines this behavior:
Implementations will evaluate each reference token against the
   document's contents and will raise an error condition if it fails to
   resolve a concrete value for any of the JSON pointer's reference
   tokens. 

Since in the spec, evaluating a reference token is defined by only the array and object cases, then evaluating a reference token against any other type will result in an error.